app/src/main/res/values/dimens.xml

```diff
@@ -3,6 +3,7 @@
   <!-- Default screen margins, per the Android Design guidelines. -->
   <dimen name="activity_horizontal_margin">16dp</dimen>
   <dimen name="activity_vertical_margin">16dp</dimen>
+  <dimen name="match_parent">-2px</dimen>
```

Comment:
why have you used -ive margin for match_parent ?

Reply:
Firstly, this is not the best industry practice but for our case it is the best solution.

Because we want to merge all files into one file but the issue is a view can be `match_parent` or `360dp` (something fixed). Now in dimens.xml file we cannot mention `match_parent` directly but the equivalent of that is `-2px` and similarly `-1px` is `wrap_content`. This `-2px` is actually not a value that will get used instead it denotes that the OS has to use `match_parent` here.